shell/lisa_shell

```diff
@@ -82,6 +82,26 @@ EOF
 export LISA_USE_VENV=${LISA_USE_VENV:-1}
 export LISA_VENV_PATH=${LISA_VENV_PATH:-"$LISA_HOME/.lisa-venv3"}
 
+function _lisa-nbextensions-setup {
+    # Check for ipywidget extensions
+    grep jupyter-js-widgets/extension \
+        .lisa-venv3/etc/jupyter/nbconfig/notebook.json \
```

Comment:
Presence of a venv cannot be assumed, and does not seem necessary here (see below).

Reply:
Why should not be assumed? Don't we always initialize a venv at lisa installation time?

Reply:
no, that depends on LISA_USE_VENV, and the path to the venv can be modified too using LISA_VENV_PATH. It is both supported and perfectly fine to run lisa-install without activating a venv first. This is necessary to have LISA be installed inside another user-managed virtualenv (that may contain other things than LISA). That is also necessary in order to be able to use a system-wide install of LISA.

Reply:
This has been removed... we don't relay on VENV anymore.